CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changes between versions
 
+## 2.0.0: Diet
+
+* added lock to StringUtil
+* added lock to EditorBuilder
+* added lock to EditorFactory
+* removed deprecated features
```

Comment:
Quite right, I should also add a BC-break warning, even if the change of the MAJOR number is a big hint.